spec/controllers/admin/booths_controller_spec.rb

```diff
@@ -0,0 +1,149 @@
+require 'spec_helper'
+
+describe Admin::BoothsController do
+
+  let(:admin) { create(:admin) }
+  let(:conference) { create(:conference) }
+  let(:booth) { create(:booth, conference: conference) }
```

Comment:
Let's define the title here, as we are changing it in the tests. That way we can explicitly know its initial value versus its changed one in the tests.